package.json

```diff
@@ -10,22 +10,23 @@
     "access": "public"
   },
   "scripts": {
-    "clean": "rm -r dist/*",
-    "build:types": "yarn tsc --emitDeclarationOnly",
+    "build": "npm run build:clean && npm run build:js",
+    "build:clean": "rm -r dist/* || true",
     "build:js": "rollup -c",
-    "build": "npm run build:types && npm run build:js",
+    "build:types": "yarn tsc --emitDeclarationOnly",
```

Comment:
You're using `npm` everywhere except here, maybe we should get rid off all the yarn stuff since it's not required.

Reply:
I was actually unable to get this running in npm for reasons I'm unsure of :/ Agree though that it should be resolved. Maybe you can assert it works for you at some point?